.travis.yml

```diff
@@ -111,12 +119,12 @@ matrix:
 
     # osx gcc
     - os: osx
-      osx_image: xcode8
+      osx_image: xcode11
```

Comment:
I would rather keep the old osx_image if possible, to notify if we break backward-compatibility.

Reply:
Okay, I'll revert and see if it works. The reason I upgraded was that brew was breaking on older xcode. But the `update: true` along might workaround this.

Reply:
Worked 🎉